input/metadata.json

```
@@ -0,0 +1,1355 @@
+              "tai_program_start_day": {
+                "type": "number",
+                "description": "TAI program start day for ED-TAI protocol -- The TAI program start day for ED-TAI protocol.",
+                "minimum": 0,
+                "default": 72,
+              },
+            }
+          },
+          "bodyweight": {
+            "type": "object",
+            "description": "Bodyweight",
+            "birth_weight_avg_ho": {
+              "type": "number",
+              "description": "Average Holstein Birth Weight (kg/head) -- The average birthweight for Holstein cows (default is 43.9)",
+              "default": 43.9,
+              "minimum": 0
+            },
+            "birth_weight_std_ho": {
+              "type": "number",
+              "description": "Holstein Birth Weight Standard Deviation (kg/head) -- The standard deviation birthweight for Holstein cows (default is 1)",
+              "default": 1,
+              "minimum": 0
+            },
+            "birth_weight_avg_je": {
+              "type": "number",
+              "description": "Average Jersey Birth Weight (kg/head) -- The average birthweight for Jersey cows (default is 35)",
+              "default": 35,
+              "minimum": 0
+            },
+            "birth_weight_std_je": {
+              "type": "number",
+              "description": "Jersey Birth Weight Standard Deviation (kg/head) -- The standard deviation birthweight for Jersey cows (default is 1)",
+              "default": 1,
+              "minimum": 0
+            },
+            "target_heifer_preg_day": {
+              "type": "number",
+              "description": "Target Heifer Pregnancy Day (days) -- The target pregnant age in days for heifers - for adjusting heifer body weight (default is 420)",
+              "default": 420,
+              "minimum": 0
+            },
+            "mature_body_weight_avg": {
+              "type": "number",
+              "description": "Average Mature Body Weight (kg/head) -- The average mature body weight of cows (default is 740.1)",
+              "default": 740.1,
+              "minimum": 0
+            },
+            "mature_body_weight_std": {
+              "type": "number",
+              "description": "Mature Body Weight Standard Deviation (kg/head) -- The standard deviation of mature body weight of cows (default is 73.5)",
+              "default": 73.5,
+              "minimum": 0
+            }
+          }
+        },
+        "from_literature": {
+          "type": "object",
+          "description": "From Literature",
+          "repro": {
+            "type": "object",
+            "description": "Literature Repro Values",
+            "preg_check_day_1": {
+              "type": "number",
+              "description": "Pregnancy Check Day 1 (days) -- Days since last insemination to check whether the animal is pregnant or not for the first time (default is 43.9).",
+              "default": 43.9,
```

This one is 32 from the json you shared with me from Yijing.
```suggestion
              "default": 32,
```